lib/xlibc-rt/check_stack.c

```diff
@@ -0,0 +1,25 @@
+#include <debug.h>
+#include "xboxkrnl/xboxkrnl.h"
+
+#ifdef DEBUG_CONSOLE
+    #define _print DbgPrint
+#else
+    #define _print debugPrint
+#endif
+
+void _xlibc_check_stack (unsigned int eax, unsigned int esp)
```

Comment:
Given that this is called from the assembly, you might want to add an explicit calling convention here

Reply:
Will do.